pkg/ntpmonitor/ntpmonitor.go

```diff
@@ -160,7 +190,7 @@ func (n *NTPMonitor) QueryNTPServer(srv string) (*ntp.Response, error) {
 		opts := ntp.QueryOptions{
```

Comment:
nit: Should `QueryNTPServer` be private, because we don't call it elsewhere?

Reply:
Yeah, looks like it isn't called anywhere else.